components/renku_data_services/users/api.spec.yaml

```diff
@@ -531,7 +531,7 @@ components:
       example: "user/my-project"
       # limitations based on allowed characters in project slugs from Gitlab from here:
       # https://docs.gitlab.com/ee/user/reserved_names.html
-      pattern: "^[a-zA-Z0-9]([_.\\-/]?[a-zA-Z0-9]+)*[a-zA-Z0-9]$"
+      pattern: "^[a-zA-Z0-9]+([_.\\-/][a-zA-Z0-9]+)*[_.\\-/]?[a-zA-Z0-9]$"
```

Comment:
```suggestion
      pattern: "^[a-zA-Z0-9]+([_.\-/][a-zA-Z0-9]+)*[_.\-/]?[a-zA-Z0-9]$"
```

Reply:
I tested what we had in the python repl and it did not complain and it seems to have worked just fine.

```
>>> re.match("^[a-zA-Z0-9]([_.\\-/]?[a-zA-Z0-9]+)*[a-zA-Z0-9]$", "test\fdf/fdsf")
>>> re.match("^[a-zA-Z0-9]([_.\\-/]?[a-zA-Z0-9]+)*[a-zA-Z0-9]$", "test/fdsf")
<re.Match object; span=(0, 9), match='test/fdsf'>
```

So it seems like we got lucky before. But we may as well fix it up now.

Reply:
Disregard this. The extra `\` is to escape the `\` in a regular python string. I got confused by this - not sure what I was thinking.